pallets/pallet-did-lookup/src/lib.rs

```diff
@@ -24,10 +24,12 @@
 
 #![cfg_attr(not(feature = "std"), no_std)]
 
+pub mod account;
+mod connection_record;
+
 pub mod default_weights;
 pub mod migrations;
```

Comment:
I wouldn't mix `pub mod` and `mod`. The style we use is to group those statements by `pub mod`, `mod`, mods with features, etc.